### PR TITLE
Fluent treelist

### PR DESCRIPTION
**Pull request by JoomFX**
related to: https://github.com/telerik/kendo-themes-private/issues/3

NOTE: The Treelist depends on the Grid styles.

### CODE REVIEW

File: packages/fluent/scss/treelist/_layout.scss [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+        .k-drop-hint {
+            transform: translate(0, -50%);
+        }
+
+        tr.k-footer td,
+        tr.k-footer-template td {
+            border-width: var( --kendo-treelist-footer-row-border-width, #{$kendo-treelist-footer-row-border-width} ) 0;
+            border-style: solid;
+            font-weight: bold;
```
Comment (epetrow) [inline packages/fluent/scss/treelist/_layout.scss:44]: Could we use --kendo-font-weight-bold here?

File: packages/fluent/scss/treelist/_layout.scss [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+        .k-status {
+            padding-inline: .6em;
+            padding-block: .4em;
+            line-height: 1.6em;
+
+            // TODO: see what this does and use a better name
+            .k-loading {
+                vertical-align: baseline;
+                margin-right: 5px;
```
Comment (epetrow) [inline packages/fluent/scss/treelist/_layout.scss:19]: Should we use a different margin here, like 4px or 6px?

**Review by epetrow — APPROVED** [review]

File: packages/fluent/scss/treelist/index.scss [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+);
+
+
+// Component
+@forward "_variables.scss";
+@use "_layout.scss" as *;
+@use "_theme.scss" as *;
+
+// Register
+@use "../core/module-system" as module;
```
Comment (TeyaVes) [inline packages/fluent/scss/treelist/index.scss:17]: add / at the end after module-system

**Review by TeyaVes — APPROVED** [review]

File: packages/fluent/scss/treelist/_layout.scss [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+                vertical-align: baseline;
+                margin-right: map-get( $kendo-spacing, 1 );
+
+                // TODO
+                display: none;
+            }
+        }
+
+        tr.k-hidden {
+            display: none;
```
Comment (TeyaVes) [inline packages/fluent/scss/treelist/_layout.scss:28]: do we need this rule? k-hidden has display: none !important

**Review by Juveniel — APPROVED** [review]

**Pull request merged**